git-notifier

```diff
@@ -296,15 +295,15 @@ def git(args, stdout_to=subprocess.PIPE, all=False):
 
     try:
         if Config.debug:
-            print >>sys.stderr, "> git " + args
```

Comment:
`%s`

Reply:
Well spotted, I've fixed this via amend/rebase.